fastlane/Fastfile

```diff
@@ -178,10 +192,10 @@ platform :ios do
       tag_name: tag,
       description: notes,
       commitish: "main",
-      upload_assets: '../Desktop/TwilioVerify.framework.zip'
+      upload_assets: '../Desktop/TwilioVerify.zip'
     )
     
-    pod_push(path: "./TwilioVerify.podspec", verbose: true)
+    pod_push(path: "./TwilioVerifySDK.podspec", verbose: true)
```

Comment:
Will this impact cocoapods?

Reply:
And SPM?

Reply:
Yeah, changed both. The pod spec & the package. 👍